raiden/transfer/channel.py

```diff
@@ -1743,7 +1780,14 @@ def send_expired_withdraws(
             break
 
         nonce = get_next_nonce(channel_state.our_state)
-        channel_state.our_state.nonce = nonce
```

Comment:
Why don't we need this line anymore?

Reply:
I brought back the line, it's my impression that we need to update the nonce if we're to emit events that use it.